artview/components/limits.py

```diff
@@ -120,7 +120,7 @@ def _default_limits(field, plot_type):
 ###############################
```

Comment:
In [L59-64](https://github.com/nguy/artview/blob/master/artview/components/limits.py#L59), we could set an aspect ratio for airborne or rhi identified files.